assets/js/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('hof/frontend/themes/gov-uk/client-js');
+const govuk = require('govuk-frontend');
+const $ = require('jquery');
+const accessibleAutocomplete = require('accessible-autocomplete');
+
+
+$('.typeahead').each(function applyTypeahead() {
```

Comment:
jquery can be replaced with vanilla js
`document.addEventListener('DOMContentLoaded', () => {
  document.querySelectorAll('.typeahead').forEach(function applyTypeahead(...
`
Then jquery can be removed as a dependency

Reply:
replaced